### ISSUE TITLE
Validate if cpp_info.name is required

### ISSUE THREAD

**Issue by uilianries** [conan-io/hooks#116]
As all package names as lower-case, but some of them use upper-case or camel-case for cmake find package, the cpp_info.name is able to generate the correct case.

This hook could validate the package name vs the cmake file in https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/

**Comment by SSE4** [comment]
obviously, it's not always required, as in most cases package name matches the `cpp_info.name` value
